challenge_1/c/xoumi/ReverseString.c

```diff
@@ -0,0 +1,27 @@
+#include<stdio.h>
+void main()
```

Comment:
if this was part of a bigger more complex program then yes I would use int I guess. I didn't really have any special reason to use void, just did~
I can most certainly use int here :)